lib/routes/workflows.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+/*
+ * @fileoverview Content system routing
+ *
+ */
+const config = require('config');
+const content = require('punchcard-content-types');
+const _ = require('lodash');
+
+const utils = require('../utils');
+const database = require('../database');
+const flow = require('../workflows/data');
```

Comment:
This should be `workflows` and it should be requiring `../workflows`; that'll give you `data`